tests/lib/journalctl.sh

```diff
@@ -35,14 +35,20 @@ check_journalctl_ready(){
 check_journalctl_log(){
     expression=$1
     shift
-    for _ in $(seq 10); do
-        log=$(get_journalctl_log "$@")
-        if echo "$log" | grep -q -E "$expression"; then
+    # Initial checks dont print the log in the debug output
+    for iter in $(seq 9); do
```

Comment:
Is there any reason for the change from `for _ in $(seq 10)` to `iter` and 9?

Reply:
good catch, forgot to revert that :)